sbomify/apps/sboms/migrations/0038_component_is_global.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 5.2.7 on 2025-11-13 00:00
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("sboms", "0037_component_contact_profile"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="component",
+            name="is_global",
+            field=models.BooleanField(
+                default=False,
```

Comment:
You have a default value set to `False` here, so none of the existing components should show up as `None` as this is a new field.